.golangci.yml

```diff
@@ -1,43 +0,0 @@
-run:
```

Comment:
doesn't seem like a part of this PR.

Reply:
reverted the configuration change - can re-approve?